staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -343,7 +350,8 @@ func validateCustomResourceConversion(conversion *apiextensions.CustomResourceCo
 // ValidateCustomResourceDefinitionSpecUpdate statically validates
 func ValidateCustomResourceDefinitionSpecUpdate(spec, oldSpec *apiextensions.CustomResourceDefinitionSpec, established bool, fldPath *field.Path) field.ErrorList {
 	requireRecognizedVersion := oldSpec.Conversion == nil || hasValidConversionReviewVersionOrEmpty(oldSpec.Conversion.ConversionReviewVersions)
-	allErrs := validateCustomResourceDefinitionSpec(spec, requireRecognizedVersion, fldPath)
+
+	allErrs := validateCustomResourceDefinitionSpec(spec, requireRecognizedVersion, conversionAndPreserveUnknownFields(oldSpec), fldPath)
```

Comment:
webhook conversion was previously alpha, I don't think we have to do this check. let's follow-up to remove this and always require `preserveUnknownFields:false` when using webhook conversion.

Reply:
Follow-up https://github.com/kubernetes/kubernetes/pull/78501